zuul-core/src/main/java/com/netflix/zuul/message/http/HttpRequestInfo.java

```diff
@@ -58,6 +58,10 @@
 
     int getOriginalPort();
 
+    default int getClientDestinationPort() {
```

Comment:
This should throw an unsupported operation exception.  This method also needs a Javadoc.

Reply:
Done.